src/main/controllers/ApplicationDetailsController.ts

```diff
@@ -44,20 +44,34 @@ export default class ApplicationDetailsController {
     const languageParam = getLanguageParam(req.url);
     const respondRedirectUrl = `/${TranslationKeys.RESPOND_TO_TRIBUNAL_RESPONSE}/${selectedApplication.id}${languageParam}`;
     const accessToken = req.session.user?.accessToken;
-    const decisionContent = await getDecisionContent(logger, selectedApplication, translations, accessToken, res);
+
+    let decisionContent;
+
```

Comment:
Personal preference but I would prefer no gap between the declaration and block where its set. Especially when its a 'let' which isn't modified again.